storage/rocksdb/ha_rocksdb.cc

```diff
@@ -5732,6 +5732,24 @@ int ha_rocksdb::create(const char *const name, TABLE *const table_arg,
     DBUG_RETURN(HA_ERR_ROCKSDB_TABLE_INDEX_DIRECTORY_NOT_SUPPORTED);
   }
 
+  if (create_info->encrypt_type.length) {
```

Comment:
"unlikely" branch annotations?

Reply:
Good catch, MyRocks currently doesn't make a lot of use of branch prediction so it has been sort of an "out of site, out of mind" thing for me, but that doesn't mean we shouldn't use it in our changes where it makes sense. I'll add some in for these and retest.

Reply:
If you wish, you could also check if my_error itself is annotated as a cold function (and not invoked on some hot path with "no error")